client/store/login/slice.ts

```diff
@@ -0,0 +1,28 @@
+import { createSlice, Reducer } from '@reduxjs/toolkit'
+import { LoginState } from './stateType'
+import { fetchUserByInvitation } from './actions/fetchUserByInvitation'
+import { acceptInvitation } from './actions/acceptInvitation'
+
+const initialState: LoginState = {}
+
+export const loginSlice = createSlice({
+  name: 'login',
+  initialState,
+  reducers: {},
+  extraReducers: (builder) => {
+    // Add reducers for additional action types here, and handle loading state as needed
+    builder.addCase(fetchUserByInvitation.fulfilled, (state, { payload }) => {
+      state.invitedUser = payload
+    })
+    builder.addCase(acceptInvitation.fulfilled, (state, { payload }) => {
```

Comment:
Should we handle error case?
```
builder
  .addCase(acceptInvitation.fulfilled, (state, { payload }) => { /* success case */ })
  .addCase(acceptInvitation.rejected, (state, { payload }) => { /* error case */ })

Reply:
Even `acceptInvitation.pending` could be handled to display user something is going on (loading?)